src/bin/leanbundle

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```

Comment:
For context, we were discussing whether this should be its own executable or part of `lean`. I believe that a separate executable that is *not* exposed by elan is the way to go. If we go with that, I'll rewrite it in Lean.